OrbitService/ProcessList.cpp

```diff
@@ -74,11 +74,21 @@ outcome::result<void, std::string> ProcessList::Refresh() {
       continue;
     }
     std::string cmdline = std::move(cmdline_file_result.value());
-    process.set_full_path(cmdline.substr(0, cmdline.find('\0')));
-
     std::replace(cmdline.begin(), cmdline.end(), '\0', ' ');
     process.set_command_line(cmdline);
 
+    // /proc/pid/exe is a link to the executable
+    const std::filesystem::path exe_path = directory_entry.path() / "exe";
+    // -n means no new line
+    const auto exe_result = LinuxUtils::ExecuteCommand(
+        absl::StrFormat("readlink -n %s", exe_path.string()));
```

Comment:
just readlink (from unistd.h) - no need to execute commands

Reply:
and probably extract it to linux utils something like GetExecutabePath(int32_t pid)

Reply:
and have GetExecutablePath call it on linux with getpid()

Reply:
done. I also filed some bugs for refactoring and added TODOs for things I did not want to touch in this PR